### ISSUE TITLE
[Docs]: Unclear installation/setup instructions for v6.4.0

### ISSUE THREAD

**Issue by Dragory** [remix-run/react-router#9326]
### What version of React Router are you using?

6.4.0

### Steps to Reproduce

Open the documentation at https://reactrouter.com/en/6.4.0

### Expected Behavior

I expected to find a dedicated **Installation** page, similar to https://reactrouter.com/en/v6.3.0/getting-started/installation for v6.3.0, with only the React Router specific installation/configuration steps.

### Actual Behavior

I only found a [full tutorial](https://reactrouter.com/en/main/start/tutorial) where the required installation steps are mixed in with instructions that are specific to the tutorial app and unrelated to React Router itself.

**Comment by arjun2010-ece** [comment]
Please fix the documentation of react router dom as its not clean ow to use the latest version there 6.4.0. I spent a lot of time figuring out how to use that.

**Comment by Demir-Utku** [comment]
Any progress on it? Also, [Main Concepts](https://reactrouter.com/en/main/start/concepts) section is outdated.

**Comment by testacode** [comment]
It's not clear for me what to install to be able to use this with react create app.
The tutorial, which is the only place that I've found tells me the below... do I need all that to use react router dom? it's super confuse that you don't even have the basic install commands on github repo. Please let me know what to install! thanks.

<img width="682" alt="image" src="https://user-images.githubusercontent.com/1852080/218581264-5cd36123-23ac-498c-af88-af6e89e1ace2.png">

**Comment by machour** [comment]
Sorry folks, our stale bot went rogue and closed this 😅
We will be revisiting this issue soon.

**Comment by ruro122020** [comment]
Hello, 
I can fix this issue and create a section for people who just want to install react-router in their own project. I was thinking of either putting it at the very top of the tutorial section or in feature overview above the client-side routing section. Will either of those work? I'm open to other suggestions as well.

**Comment by u83ism** [comment]
Despite the comprehensive documentation, I was incredibly surprised to find that this item is missing. Upon careful observation, I understand that react-dom-router is likely necessary, but...

Considering the current landscape of front-end frameworks, it is reasonable to assume that the majority of those reading this library's tutorial are working on moderately sized React applications with multiple screens, adopting this library at a later stage. (New projects use Next.js.) Therefore, I believe prioritizing this section and placing it at the top would be beneficial.

**Comment by sovetski** [comment]
I have the same problem here, it doesn't make sense to put a tutorial to show how to install/use a library. A good old “Getting Started” will be fine and everyone will be content. It's a shame there's no progress on this.

**Comment by ryanflorence** [comment]
Yeah honestly not sure we missed that, probably meant to get back to it and then just never did.

The v7 docs have way better getting started sections with their own installation guides as step 1.
